src/index.ts

```diff
@@ -18,12 +18,14 @@ export * from './hooks'
 
 import {historyObj, changeLocation} from './reducers/router'
 export {
+    applyParams,
```

Comment:
We probably shouldn't expose this helper:
1) It's really tiny, even counting two nested functions it's only 12 lines of trivial code, it's easy to implement from the client side.
2) It is a generic helper not connected to Tesler functionallity in any way. We have a number of generic helpers and utilities in our code base related to urls, strings, dates, etc. We can't just expose all of them just because client app requires similiar functionallity and we have it implemented. There is a public API of tesler-ui and as soon as something exposed and becomes a part of this public API we have to consider semantic versioning for this helpers, meaning we are no longer free to change their contract or behavior.